frontend/store/user.js

```diff
@@ -172,9 +172,9 @@ export const actions = {
     const endTime = moment(startTime).add(5, 'm').format('DD-MM-YYYY HH:mm:ss')
     commit('setRestingPeriod', endTime)
   },
-  initQuestionnaire({commit}) {
+  initQuestionnaire({commit}, firstQuestionnaireEverDate) {
```

Comment:
Firstquestionnaireever can be saved in questionnaire instead, use setquestionnaire actions.  Don't forget to add it to history variable in store/user.js